dev/radar/appsettings.json

```diff
@@ -0,0 +1,154 @@
+﻿{
+  "Runtime": {
+    "Mode": "FULL"
+  },
+  "Jwt": {
+    "SigningKey": "LEAF_JWT_KEY",
+    "Password": "LEAF_JWT_KEY_PW",
+    "Certificate": "LEAF_JWT_CERT",
+    "Issuer": "urn:leaf:iss:dev.leaf.tld"
```

Comment:
Will this need to change for production?

Reply:
I don't *think* so - as far as I can tell this is an arbitrary string used to sign the JWT Leaf uses internally. I may need to change it for some of the later auth work (integrating with oauth2-proxy)